pom.xml

```diff
@@ -4,7 +4,7 @@
 	<parent>
 		<groupId>org.springframework.boot</groupId>
 		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>2.7.7</version>
+		<version>2.7.9</version>
```

Comment:
Good change

Reply:
Thanks, though it would be nice to use the latest version :-*